CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## New Features
 
 ## Improvements
+- PR #3270: cuML's estimator Base class for preprocessing models
```

Comment:
Please remove this changelog entry from your PR. Manual changelog entries are no longer required.

Reply:
Thanks for reminding me